canonicalwebteam/blog/django/views.py

```diff
@@ -86,6 +90,68 @@ def group(request, slug, template_path):
     return render(request, template_path, context)
 
 
+def archives(request, template_path="blog/archives.html"):
+    try:
+        page_param = request.GET.get("page", default="1")
+        category_param = request.GET.get("category", default="")
+        group_param = request.GET.get("group", default="")
+        month_param = request.GET.get("month", default="")
```

Comment:
For `month` and `year` you could probably just try to `int` cast here since we are broadly handling the exceptions?

Reply:
This might break when year or month are `""`(empty string.)